src/VisualStudio/Core/Impl/CodeModel/AbstractCodeModelService.cs

```diff
@@ -490,18 +487,12 @@ public void Rename(ISymbol symbol, string newName, Workspace workspace, ProjectC
             var newSolution = Renamer.RenameSymbolAsync(oldSolution, symbol, newName, oldSolution.Options).WaitAndGetResult_CodeModel(CancellationToken.None);
             var changedDocuments = newSolution.GetChangedDocuments(oldSolution);
 
-            // Notify third parties of the coming rename operation and let exceptions propagate out
-            _refactorNotifyServices.TryOnBeforeGlobalSymbolRenamed(workspace, changedDocuments, symbol, newName, throwOnFailure: true);
-
             // Update the workspace.
             if (!workspace.TryApplyChanges(newSolution))
             {
                 throw Exceptions.ThrowEFail();
             }
 
-            // Notify third parties of the completed rename operation and let exceptions propagate out
-            _refactorNotifyServices.TryOnAfterGlobalSymbolRenamed(workspace, changedDocuments, symbol, newName, throwOnFailure: true);
-
             RenameTrackingDismisser.DismissRenameTracking(workspace, changedDocuments);
```

Comment:
This can go away too, right?

Reply:
Not yet, dismissing rename tracking should be separate imo. Not sure if preference is to keep the original issue open or file a new one, but I want to do it in a separate change